xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/transactions/IssuedCurrencyAmount.java

```diff
@@ -32,8 +37,8 @@
   String currency();
```

Comment:
Maybe this is worth a separate issue, but maybe we could deprecate this in favor of a `BigDecimal` typed field.  I'm assuming there is a way to configure Jackson to deserialize scientific notation correctly, which would make this field much more useful.

Reply:
I like that in principle because using a number here seems more "correct" -- although the one downside of that is that a developer couldn't easily use a `String` here in their own code (they would have to translate to a BigDecimal). Is that a problem?

@nkramer44 I'm curious to hear more about how you use this field in your day-to-day development around issued currencies. Any thoughts there?

Reply:
In any event, let's create an issue for this and capture the conversation there?

Reply:
There's also this new [Discussions](https://github.com/XRPLF/xrpl4j/discussions) thing from GH -- might be an interesting first topic.

Reply:
Started a discussion thread here: https://github.com/XRPLF/xrpl4j/discussions/72